version-bump/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM python:3-slim
-
-ADD . /app
+FROM python:3-slim AS builder
 
 WORKDIR /app
+RUN pip3 install --no-cache-dir pyyaml --target=.
+ADD ./main.py .
 
-RUN pip3 install lxml --target=/app
-RUN pip3 install pyyaml --target=/app
+FROM gcr.io/distroless/python3-debian12
```

Comment:
Needed to use the container with root, as in rootless, the python code couldn't access the files in the GitHub action @aj-bw  @mandreko-bitwarden